### PR TITLE
update rocksdb, add erlang/otp 21, 22 support

### PR DESCRIPTION
**Pull request by getong**

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -2,7 +2,7 @@
 {deps,
  [
   {sext, "1.4.1"},
-  {rocksdb,"0.14.0"}
+  {rocksdb,"0.19.0"}
```
Comment (benoitc) [inline rebar.config]: you should use the 0.20.1 also

Comment (getong) [reply rebar.config]: update to 0.20.1

**Comment by getong** [comment]
Add travis test code.

File: rebar.config (hunk 1 above)
Comment (getong) [reply rebar.config]: the aternity team update it to 0.24.0, it should be resolved.

**Comment by benoitc** [comment]
maybe that can be updated to use rocksdb 0.26 : https://gitlab.com/barrel-db/erlang-rocksdb/tags/0.26.0 This will allows you to reuse the installation of rocksdb on your system (and then speed the build). cc @uwiger @getong

**Comment by uwiger** [comment]
No longer relevant.

**Pull request closed without merging**